popup.html

```diff
@@ -43,23 +43,19 @@
         </label>
       </fieldset>
       <fieldset>
-        <legend>Bibliogram instances</legend>
-        <select id="bibliogram-instances" name="bibliogram-instances"></select>
-        <input
-          name="bibliogram-instance"
-          type="text"
-          placeholder="Custom domain"
-        />
+        <legend>Scribe instances</legend>
+        <select id="scribe-instances" name="scribe-instances"></select>
+        <input name="scribe-instance" type="text" placeholder="Custom domain" />
         <button class="add-custom" type="button">+</button>
         <label class="switch">
-          <input id="switch-bibliogram" type="checkbox" checked />
+          <input id="switch-scribe" type="checkbox" checked />
           <span class="slider"></span>
         </label>
       </fieldset>
       <fieldset>
-        <legend>Scribe instances</legend>
-        <select id="scribe-instances" name="scribe-instances"></select>
-        <input name="scribe-instance" type="text" placeholder="Custom domain" />
+        <legend>Quetre instances</legend>
+        <select id="quetre-instances" name="quetre-instances"></select>
+        <input name="quetre-instance" type="text" placeholder="Custom domain" />
         <button class="add-custom" type="button">+</button>
         <label class="switch">
           <input id="switch-scribe" type="checkbox" checked />
```

Comment:
```suggestion
          <input id="switch-quetre" type="checkbox" checked />
```

Reply:
Ok, Thanks for catching that. Sorry about such a simple miss.